Makefile

```diff
@@ -2,6 +2,9 @@ xcodeproj:
 	@command -v xcodegen >/dev/null 2>&1 || { echo >&2 "Required tool missing: XcodeGen. Try 'brew install xcodegen' perhaps?"; exit 1; }
```

Comment:
should we run `carthage` before running this?

Reply:
The order doesn't matter, though if we call `xed` we should change it.

Reply:
but `make carthage` must be called before we can use the workspace right? should we combine them both in a `make bootstrap` then?

Reply:
Yeah can do!

Reply:
Done.